Resources/Locale/en-US/deltav/flavors/flavor-profiles.ftl

```diff
@@ -29,6 +29,8 @@ flavor-complex-blellow = like an impossible color
 flavor-complex-candy-strawberry = like strawberries
 flavor-complex-candy-bubblegum = like bubble gum
 flavor-complex-double-ice-cream = like ice cream, twice
+flavor-complex-drgibbbloodred = like severe malpractice
+
```

Comment:
why

Reply:
Its a joke, because its Dr Gibb made with blood, which it's current flavour is "tastes like malpractice" and well, if you do severe malpractice, wouldnt that result in blood :blunt: